src/utils/image-loading.ts

```diff
@@ -137,7 +141,6 @@ export function loadLdr(url: string): Promise<LdrImage> {
                 reject(new Error(`Failed to load image '${url}': ${error}`));
             }
         };
-        image.src = url;
     });
 }
 
```

Comment:
Note I had rewritten this file as a typescript file, then copied the js output back to this file, so there are some things changed here we may want to change back, like const -> var, it should be const. Just noting that.